semantic/src/site/globals/site.variables

```diff
@@ -164,3 +164,6 @@
 
 // Box shadow -- used in Login form
 @basicBoxShadow: rgb(204, 201, 201) 0px -3px 10px 0px;
+
+// Box shadow -- used in Dashboard 
+@templateBoxShadow: 0 1px 3px 0 #d4d4d5, 0 0 0 1px #d4d4d5;
```

Comment:
Good idea to defined this colour "#d4d4d5" in some variable

Reply:
b71df009f78493415b2a33e96c17bb96a0be4652